sklearn/preprocessing/tests/test_encoders.py

```diff
@@ -687,3 +705,63 @@ def test_one_hot_encoder_warning():
     enc = OneHotEncoder()
     X = [['Male', 1], ['Female', 3]]
     np.testing.assert_no_warnings(enc.fit_transform, X)
+
+    enc = OneHotEncoder(drop=['def', 3, 56])
```

Comment:
what's this got to do with testing the warning?

Reply:
I think I accidentally deleted the header line for the next test!